docs/reference/connection.md

```diff
@@ -3,14 +3,18 @@
 [Connection](#) module will used as a connector to connect to kintone Rest API
 
 ## Constructor
-
+**Declaration**
+```
+public Connection(String domain, Auth auth)
+public Connection(String domain, Auth auth, int guestSpaceID)
+```
 **Parameter**
 
-| Name| Type| Required| Description |
-| --- | --- | --- | --- |
-| domain | String | yes | The Domain name or FQDN
-| auth | [Auth](../authentication) | yes | The authentication object
-| guestSpaceID | Integer | (optional) | The guest space id. Use this parameter to connect to kintone guest space.
+| Name| Description |
+| --- | --- |
+| domain |The Domain name or FQDN
+| auth | The authentication object ([Auth](../authentication))
+| guestSpaceID | The guest space id. This parameter is used for connect to kintone guest space.
```

Comment:
> The guest space id. This parameter is used for connect to kintone guest space.
The guest space id. This parameter is used for connection to kintone guest space.
or
The guest space id. This parameter is used to connect to kintone guest space.
or
The guest space id. Use this parameter to connect to kintone guest space.